python/pyarrow/tests/test_compute.py

```diff
@@ -229,7 +229,7 @@ def test_sum_chunked_array(arrow_type):
 
     arr = pa.chunked_array((), type=arrow_type)
     assert arr.num_chunks == 0
-    assert pc.sum(arr).as_py() is None  # noqa: E711
+    assert pc.sum(arr).as_py() == 0
```

Comment:
This `sum empty array to 0` is a bit surprising to me. But looks pandas is doing the same thing.

Reply:
Reintroduced.